### PR TITLE
SONARIAC-1598 Add Micronaut checks and visitor with ToDo comment and Parsing rules

### PR DESCRIPTION
**Pull request by rudy-regazzoni-sonarsource**

### CODE REVIEW

File: iac-extensions/jvm-framework-config/src/main/java/org/sonar/iac/jvmframeworkconfig/checks/springconfig/AbstractSensitiveKeyCheck.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.iac.jvmframeworkconfig.checks;
+package org.sonar.iac.jvmframeworkconfig.checks.springconfig;
```
Comment (jonas-wielage-sonarsource) [inline iac-extensions/jvm-framework-config/src/main/java/org/sonar/iac/jvmframeworkconfig/checks/springconfig/AbstractSensitiveKeyCheck.java:20]: I believe `spring` and `micronaut` would be sufficient as a post fix to the package name, as we already have `config` in the name

File: iac-extensions/jvm-framework-config/src/main/java/org/sonar/iac/jvmframeworkconfig/checks/springconfig/ExcessiveFileUploadSizeLimitCheck.java [hunk 2]
```diff
@@ -17,12 +17,14 @@
...
  */
-package org.sonar.iac.jvmframeworkconfig.checks;
+package org.sonar.iac.jvmframeworkconfig.checks.springconfig;
 
 import java.util.Locale;
 import java.util.Set;
 import java.util.regex.Pattern;
+
 import javax.annotation.CheckForNull;
+
```
Comment (jonas-wielage-sonarsource) [inline iac-extensions/jvm-framework-config/src/main/java/org/sonar/iac/jvmframeworkconfig/checks/springconfig/ExcessiveFileUploadSizeLimitCheck.java:27]: It looks like you have the wrong settings for imports, there should not be a line break in between 
https://github.com/SonarSource/sonar-developer-toolset

**Review by jonas-wielage-sonarsource — APPROVED** [review]
LGTM! Please have a look at my suggestions

File: iac-extensions/jvm-framework-config/src/main/java/org/sonar/iac/jvmframeworkconfig/checks/springconfig/ExcessiveFileUploadSizeLimitCheck.java (hunk 2 above)
Comment (rudy-regazzoni-sonarsource) [reply iac-extensions/jvm-framework-config/src/main/java/org/sonar/iac/jvmframeworkconfig/checks/springconfig/ExcessiveFileUploadSizeLimitCheck.java:27]: Ahh, sorry it happened again.
I have good settings, but dunno why my local IaC project doesn't want to accept them.
I had to clone the repo again to finally make it work, but I mistakenly worked on the previous clone.

Comment (jonas-wielage-sonarsource) [reply iac-extensions/jvm-framework-config/src/main/java/org/sonar/iac/jvmframeworkconfig/checks/springconfig/ExcessiveFileUploadSizeLimitCheck.java]: I will create a ticket to add import formatting to spotless, we shouldn't rely on our individual machine here.

**Pull request merged**